docs/source/reference/config-user-env.md

```diff
@@ -1,21 +1,22 @@
+![jupyterhub](https://jupyter.org/assets/homepage/hublogo.svg)
+
```

Comment:
I don't think this is needed since our documentation theme will always put our logo in the top menu bar anyway